ocpmodels/common/relaxation/optimizers/lbfgs_torch.py

```diff
@@ -153,28 +175,34 @@ def determine_step(dr):
 
         e, f = self.get_forces()
         f = f.to(self.device, dtype=torch.float64)
-        r = self.atoms.pos.to(self.device, dtype=torch.float64)
+        r = self.batch.pos.to(self.device, dtype=torch.float64)
 
         # Update s, y and rho
         if iteration > 0:
-            s0 = (r - r0).flatten()
-            y0 = -(f - f0).flatten()
+            s0 = r - r0
+            y0 = -(f - f0)
             s.append(s0)
             y.append(y0)
-            rho.append(1.0 / torch.dot(y0, s0))
+            rho.append(1.0 / _batched_dot(y0, s0))
 
         loopmax = min(self.memory, iteration)
         alpha = f.new_empty(loopmax)
```

Comment:
Shouldn't `alpha` be of size `loopmax x batch_size`? Since in L193 we assign a `batch_size` vector in each iteration of `loopmax`.

Reply:
Fixed this.